SpatialGDK/Source/SpatialGDK/Public/Schema/ServerRPCEndpoint.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved	
+
+#pragma once	
```

Comment:
I feel like we can just have one RPCEndpoint.h file and make both endpoints sit in there. Just a thought, since they both share the same includes.

Reply:
We decided to keep both since they have different Spatial IDs.